ci/docker/release-cpu.yaml

```diff
@@ -24,3 +24,6 @@ spack:
     all:
       variants:
         - 'build_type=Release'
+    pika:
+      require:
+        - '@git.fake-0.29.0=0.29.0'
```

Comment:
Remove before merging:
```suggestion
```
`fake-0.29.0` is a branch with the pika version bumped to 0.29.0 to check the ifdefs including/excluding the setting of the number of handles.